inc/agent.class.php

```diff
@@ -42,7 +42,10 @@ class PluginFlyvemdmAgent extends CommonDBTM implements PluginFlyvemdmNotifiable
    const ENROLL_INVITATION_TOKEN = 1;
    const ENROLL_ENTITY_TOKEN     = 2;
 
-   const DEFAULT_TOKEN_LIFETIME  = "P7D";
+   const DEFAULT_TOKEN_LIFETIME  = 'P7D';
+
+   const MINIMUM_ANDROID_VERSION = '2.0';
+   const MINIMUM_APPLE_VERSION = '1.0';
```

Comment:
introduce constants. They are also used for the unit tests to reduce maitnenance.